web/blog/2024-09-24-why-your-emails-arent-getting-delivered-and-how-to-fix-this-problem.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Why Your Emails Aren't Getting Delivered and How to Fix This Problem"
+authors: [milica]
+image: /img/email/email-cover.jpg
+tags: [webdev, tips, emails]
+---
+import ReactPlayer from 'react-player'
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ImgWithCaption from './components/ImgWithCaption'
+
+If you’re new to [Wasp](https://wasp-lang.dev/), and you’ve just built and deployed your app, it’s possible that your emails aren’t reaching your users. Don’t be too alarmed. This is a common issue that affects many developers who have newly registered domains.
```

Comment:
The advice below isn't specific for Wasp, right? I'd then also make the intro more general, for any SaaS builder / indie hacker. I'd start with something like a problem statement, e.g. "You've been hacking at your newest SaaS idea for a couple of weeks and you finally released it to the world. But, soon, you start getting complaints - the transactional emails your app sends (e.g. for signup or X/Y/Z) aren't getting delievered! Do not worry - that's a common problem ..."

Reply:
What was the problem that Wasp users encountered? We can use that use case probably.

Reply:
Yup, you're right, I'll clarify the use case we had and make the first few lines more neutral :)